qemu/tests/block_hotplug.py

```diff
@@ -121,33 +123,43 @@ def get_new_disk(disk1, disk2):
 
             device_list.append(device)
             if not new_disks:
-                raise exceptions.TestFail("Cannot find new disk after hotplug.")
+                test.fail("Cannot find new disk after hotplug.")
 
             if params.get("need_plug") == "yes":
                 disk = new_disks[0]
             else:
                 disk = new_disks[num]
 
+            session = vm.wait_for_login(timeout=timeout)
+            if params.get("format_disk", "no") == "yes":
+                if i == 0:
+                    error_context.context("Format disk", logging.info)
+                    utils_misc.format_windows_disk(session,
```

Comment:
why only format windows disk here ? how about linux guest?

Reply:
Hi coli, 
For linux guest, the r/w operation run on the bare disk, it no need format; 
For windows, we have a request to run iozone r/w operation on the new hotpluged disk, iozone cannot run on a bare disk, so it need format firstly.